README.md

```diff
@@ -1,104 +1,153 @@
+  ```
+  ### Question #3
 
-### Question #5
+  Given the following front-end JS model, add an instance method for all pandas called `eat_bamboo`. Calling this method should increase that panda's value for `num_bamboo_eaten` by 1.
 
-Which of the following statements will work, assuming jQuery is loaded?
+  ```js
 
-Select all that apply:
-```
-[] `$(".post").css("background", "peachpuff")`
-[] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.querySelectorAll(".post")[0].innerHTML`
-[] `document.querySelectorAll(".post").innerHTML`
-```
+  var Panda = function(name, age) {
+    this.name = name;
+    this.age = age;
+    this.num_bamboo_eaten = 0;
+  }
+  ```
+  Your Answer:
+  ```js
 
-### Question #6
+  Panda.prototype.eat_bamboo = function() {
+    this.num_bamboo_eaten = num_bamboo_eaten + 1;
+  };
 
-Using jQuery, add an event listener for clicks on the button with the id
-`greeting`. When the event happens, the code should append a paragraph to the
-body that says "hello".
 
-Your Answer:
-```js
-// your code here
-```
+  ```
 
-### Question #7
+  ### Question #4
 
-Define a function called `doSomething`. It should take one argument, called
-`thingToDo`. When called, `doSomething` should invoke the `thingToDo` function. Demonstrate calling `doSomething` while passing in an argument.
+  Describe the importance of using object-oriented programming.
 
-Your Answer:
-```js
-// write code here
-```
+  Your Answer:
+  ```js
+  object-oriented programming helps programmers to organize code into parts with its own usage or characters but always call refer or manipulate at any time without repeating the same code in each function.
 
-### Question #8
 
-Once in Vanilla JS, and once in jQuery, write a function that adds an event listener for when a button with a class of "submit-quiz" is clicked. The event alerts a user "Great Job on Quiz 4!".
+  ```
 
-Your Answer:
-```js
-// write code here
-```
+  ## jQuery
+
+  ### Question #5
+
+  Which of the following statements will work, assuming jQuery is loaded?
+
+  Select all that apply:
+  ```
+  [x] `$(".post").css("background", "peachpuff")`
+  [] `$(".post").innerHTML`
+  [x] `$(".post").html()`
+  [x] `document.querySelectorAll(".post")[0].innerHTML`
+  [] `document.querySelectorAll(".post").innerHTML`
+  ```
+
+  ### Question #6
+
+  Using jQuery, add an event listener for clicks on the button with the id
+  `greeting`. When the event happens, the code should append a paragraph to the
+  body that says "hello".
+
+  Your Answer:
+  ```js
+  $("#greeting").on("click", function(){
+    $("body").append("<p>hello</p>")
+  })
+
+  ```
+
+  ### Question #7
+
+  Define a function called `doSomething`. It should take one argument, called
+  `thingToDo`. When called, `doSomething` should invoke the `thingToDo` function. Demonstrate calling `doSomething` while passing in an argument.
+
+  Your Answer:
+  ```js
+  function doSomething(thingToDo){
+    thingToDo(){
+      console.log("Hello");
```

Comment:
Almost! You just need `thingToDo()`. Also, you need to "Demonstrate calling `doSomething` while passing an argument."